services/scheduler/src/main/java/org/mule/service/scheduler/internal/DefaultSchedulerService.java

```diff
@@ -85,6 +88,7 @@
   private org.quartz.Scheduler quartzScheduler;
 
   private volatile boolean started = false;
+  private List<Scheduler> activeSchedulers = synchronizedList(new ArrayList<>());
```

Comment:
Also, there is a callback provided so these are removed when the scheduler is stopped,